src/Interpreters/ClientInfo.h

```diff
@@ -25,6 +25,7 @@ class ClientInfo
     {
         TCP = 1,
         HTTP = 2,
+        GRPC = 3,
```

Comment:
What about MySQL and PostgreSQL interfaces?

Reply:
https://github.com/ClickHouse/ClickHouse/pull/17437